sphinxcontrib/openapi/cmdline.py

```diff
@@ -0,0 +1,77 @@
+
+from __future__ import unicode_literals
+
+import argparse
+import collections
+import io
+import logging
+
+import yaml
+
+from sphinxcontrib.openapi import openapi20
+from sphinxcontrib.openapi import openapi30
+
+
+class _YamlOrderedLoader(yaml.SafeLoader):
+    pass
+
+
+_YamlOrderedLoader.add_constructor(
+    yaml.resolver.BaseResolver.DEFAULT_MAPPING_TAG,
+    lambda loader, node: collections.OrderedDict(loader.construct_pairs(node))
+)
```

Comment:
Rather than redefining this, couldn't we just import it from 'sphinxcontrib.openapi.directive'? You can keep it private since we don't necessarily want anyone else using it.

Reply:
Done.